### ISSUE TITLE
docs: Accessibility Section in the Documentation of ALL Components

### ISSUE THREAD

**Issue by DiogoFilipeG** [goetzrobin/spartan#877]
### Which scope/s are relevant/related to the feature request?

Don't know / other

### Information

Component documentation should have a section talking about Accessibility and the complience with https://www.w3.org/WAI/ARIA/apg/patterns/

### I would be willing to submit a PR to fix this issue

- [ ] Yes
- [x] No
